assets/section-main-product.css

```diff
@@ -45,6 +46,10 @@
     padding: 0 8rem 0 0;
   }
 
+  .product--right .product__media-list {
+    margin-bottom: 2rem;
+  }
```

Comment:
For context, this was added as a quick fix for: https://github.com/Shopify/dawn/issues/1903 and I specifically added this to override this: https://github.com/Shopify/dawn/pull/1110/files#diff-00b03bd62b155daa89082ba52094c182e46422a9e28cd3ec601b3a2291b52718

>Maybe we could be more specific on the classes

Can you please elaborate on this more? Specificity?

**I'll just add this just in case you were referring to it**: I thought about adding a modifier to `product__media-list` (for the duplicated right desktop media position) and add the following CSS instead:

```css
.product--right .product__media-list--right {
  margin-bottom: 2rem;
}
```

... or something along those lines but thought this works just as well. Maybe I could add a CSS comment to explain what this is doing?

Reply:
Oh I guess we're applying the same `marign-bottom` on the `.grid` class when the medias are on the left. I was just thinking that this margin seems mainly helpful when we have the thumbnails showing but otherwise doesn't seem necessary. 
But if it's already using it on the `ul.grid` element on the left then that's fine to keep it as is 👍 🙂